src/nextjournal/clerk/parser.cljc

```diff
@@ -48,7 +48,13 @@
 (defn visibility-marker? [form]
   (and (map? form) (contains? form :nextjournal.clerk/visibility)))
 
+(def doc-settings
+  "Settings allowed in the namespace's metadata map."
+  #{:nextjournal.clerk/doc-css-class})
```

Comment:
If we introduce this, I think `:error-on-missing-vars` and `:toc-visibility` are `doc-settings` as well?

Reply:
Yes, maybe. I first thought the names in the list are those whose values are parsed as-they-are and propagated to render fns, but that's wrong. 

I'd also assume the keywords in that list to be user-facing settings. So in that case we'd need to change the `:n.c/toc => :n.c/toc-visibility`.